plugins/wfsnav.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id":"154",
+    "title":"WFSnav: Smooth transition to blocks",
+    "image":"154-wfsnav.js.jpg",
+    "description":"WFScroll - Smooth transition to blocks - Class on the active menu item - Closing the mobile menu with further moving",
```

Comment:
Keep it simple and explanatory: "Allow a quality navigation experience through smooth scroll to blocks, active state for menu item and auto closing its mobile version: